### DISCUSSION TITLE
Windows Access denied

### DISCUSSION THREAD

**Discussion by Adhithya-Pinghley** [3mora2/WPP_Whatsapp#9] [General]
I have done pip install wpp_whatsapp and run this code in one system, in which it runs without any problem. But I'm trying to run it on a different system, which throws the below attached error, which apparently says that there is some permission error- I suppose it requires some higher privileges. So I tried to give permission using runas administrator command, but unfortunately I forgot the password. Is there any way I can bypass this issue?? If you need any more information I'm ready to provide  

[Error.txt](https://github.com/3mora2/WPP_Whatsapp/files/12050441/Error.txt)
[whatsapptestfile.txt](https://github.com/3mora2/WPP_Whatsapp/files/12050442/whatsapptestfile.txt)

**Comment by eletrofss** [comment]
do the sethc trick using a linux usb drive then call control userpasswords2 and redefine the password.
